torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6917,6 +6932,23 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            # the gradient check for complex fails.
            backward_dtypes=floating_types_and(torch.float16, torch.bfloat16),
            ),
+    OpInfo('attn',
+           dtypes=floating_types(),
+           sample_inputs_func=sample_inputs_attn,
+           skips=(
+               SkipInfo('TestCommon', 'test_out'),
+               SkipInfo('TestOpInfo', 'test_unsupported_dtypes'),
+           )),
+]
+
+op_db: List[OpInfo] = [
```

Comment:
Is this so that you only test this one when running tests locally? :D
When running your test, you can do `python test_ops.py -k "attn"` to only run the tests that have `attn` in their name. Allowing you to filter these ones explicitely.

Reply:
Oops, yes this was for local tests. Good to know the `-k` flag for the future!